jsons/1.12-rel.json

```diff
@@ -35,9 +35,28 @@
       "serverreq":true
     },
     {
-      "name": "jline:jline:2.13",
+      "name": "net.minecraftforge:log4j-server-fix:2.8.1",
       "url" : "http://files.minecraftforge.net/maven/",
-      "checksums" : [ "2d9530d0a25daffaffda7c35037b046b627bb171" ],
+      "checksums" : [ "TODO" ],
```

Comment:
Needs to be built using https://gist.github.com/Minecrell/0d4ccfab75eee044a654b196e61c93c8 (put it in a directory called `log4j-server-fix`, then run `gradle build`) and uploaded to the Forge maven to make this work.